src/lib/entities/podcast/ui/podcast-display.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  import { H1, s } from '$lib/shared/ui';
+  import type { Podcast } from '$lib/shared/api';
+
+  export let podcast: Podcast;
+</script>
+
+<div class="flex flex-row gap-4 p-2">
+  <img class="w-24 h-24 rounded-xl" src={podcast.coverUrl} alt="" />
+  <div class="flex flex-col gap-1 text-xs pt-2">
+    <H1>{podcast.title}</H1>
+    <p>{podcast.author}</p>
+    <p>
+      {podcast.episodes.length} episode{s(podcast.episodes.length)} / <slot name="download-all" />
```

Comment:
According to Feature-Sliced Design, we cannot reference `features` from `entities`, and downloading is a feature (because it involves the user interacting with the UI). I already had a conversation with Max and Khaled, selling them my vision of FSD, perhaps you and I could have it as well :)

Reply:
I think that would be helpful indeed since the names of the slices do not really stick in my head at first, they don't feel very intuitive